docs/excel/excel-add-ins-ranges-set-get.md

```diff
@@ -47,6 +47,95 @@ Excel.run(function (context) {
+
+In the following screenshot, the used range is the table with values in each cell, **C5:F12**. The empty cells outside this table are outside the used range.
+
+![A table with data from C5:F12 in Excel.](../images/excel-ranges-used-range.png)
+
+### Select the cell at the edge of the current used range
+
+The following code sample shows how use the `Range.getRangeEdge` method to select the cell at the furthest edge of the current used range, in the direction up. This action matches the result of using the Ctrl+Arrow key keyboard shortcut while a range is selected.
+
+```js
+Excel.run(function (context) {
+    // Get the selected range.
+    var range = context.workbook.getSelectedRange();
+
+    // Specify the direction with the `KeyboardDirection` enum.
+    var direction = Excel.KeyboardDirection.up;
+
+    // Get the active cell in the workbook.
+    var activeCell = context.workbook.getActiveCell();
+
+    // Get the top-most cell of the current used range.
+    // This method acts like the Ctrl+Arrow key keyboard shortcut while a range is selected.
+    var rangeEdge = range.getRangeEdge(
+      direction,
+      activeCell
+    );
+    rangeEdge.select();
+
+    return context.sync();
+}).catch(errorHandlerFunction);
+```
+
+#### Before selecting the cell at the edge of the used range
+
+The following screenshot shows a used range and a selected range within the used range. The used range is a table with data at **C5:F12**. Inside this table, the range **D8:E9** is selected. This selection is the *before* state, prior to running the `Range.getRangeEdge` method.
+
+![A table with data from C5:F12 in Excel. The range D8:E9 is selected.](../images/excel-ranges-used-range-d8-e9.png)
+
+#### After selecting the cell at the edge of the used range
+
+The following screenshot shows the same table as the preceding screenshot, with data in the range **C5:F12**. Inside this table, the range **D5** is selected. This selection is *after* state, after running the `Range.getRangeEdge` method to select the cell at the edge of the used range in the up direction.
+
+![A table with data from C5:F12 in Excel. The range D5 is selected.](../images/excel-ranges-used-range-d5.png)
+
+### Select all cells from current range to furthest edge of used range
+
+The following code sample shows how use the `Range.getExtendedRange` method to to select all the cells from the currently selected range to the furthest edge of the used range, in the direction down. This action matches the result of using the Ctrl+Shift+Arrow key keyboard shortcut while a range is selected.
```

Comment:
I mentioned this previously but I'll leave it up to you. I think you should describe which arrow you mean since there are actually four on the keyboard.

From the style guide: Use sentence capitalization to refer to a specific arrow key: the Left arrow key, the Right arrow key, the Up arrow key, or the Down arrow key. It's OK to use arrow key as a general term for any single arrow key. Include the and key in references to a specific arrow key except in key combinations or key sequences.

Reply:
I'll fix this throughout. Thanks!